src/components/attendee/ProfileLinkIcon/ProfileLinkIcon.module.scss

```diff
@@ -0,0 +1,18 @@
+@use "scss/attendee/space";
+@use "scss/attendee/border";
+@use "scss/attendee/theme";
+
+.linkWrapper {
+  width: 5%;
```

Comment:
Is this responsive friendly?

Reply:
just added a `10%` for everything lower than our `$md` ~768px, looks way better, thanks
<img width="219" alt="image" src="https://user-images.githubusercontent.com/56254806/163201320-cac22b94-ab53-411d-8800-3834eafd5996.png">

Reply:
[04f5003](https://github.com/sparkletown/sparkle/pull/3027/commits/04f5003da32c5d46ad3c388886090803250bdac9)